src/components/SignInForm.tsx

```diff
@@ -0,0 +1,86 @@
+import Image from "next/image";
+import SignInButton from "./SignInButton";
+
+export default function SignInForm() {
+  return (
+    <>
+      <div className="flex min-h-screen flex-1 flex-col justify-center px-6 py-12 lg:px-8">
+        <div className="sm:mx-auto sm:w-full sm:max-w-sm">
+          <Image
+            alt="PodCodar Llama"
+            className="mx-auto h-10 w-auto"
+            height={100}
+            priority
+            src="/images/logo.svg"
+            style={{height: "100px"}}
+            width={100}
+          />
+          <h2 className="mt-10 text-center text-2xl font-bold leading-9 tracking-tight text-white">
+            Sign in to your account
+          </h2>
+        </div>
+
+        <div className="mt-10 sm:mx-auto sm:w-full sm:max-w-sm">
+          <form
+            action="#"
+            className="space-y-6 mb-5"
+            method="POST"
+          >
+            <div>
+              <label className="block text-sm font-medium leading-6 text-white" htmlFor="email">
```

Comment:
I think we can remove the inputs for email and password, as we only will allow social login for the MVP